circle.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
     - sudo apt-get install python-numpy python-scipy python-dev python-matplotlib
     - sudo apt-get install python-nose python-coverage
     - sudo apt-get install python-sphinx
+    - pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
```

Comment:
We want to use wheels for alleviating computation
Le 3 nov. 2015 19:48, "Manoj Kumar" <notifications@github.com> a écrit :

> In circle.yml
> <https://github.com/scikit-learn/scikit-learn/pull/5492#discussion_r43788755>
> :
>
> > @@ -13,6 +13,7 @@ dependencies:
> >      - sudo apt-get install python-numpy python-scipy python-dev python-matplotlib
> >      - sudo apt-get install python-nose python-coverage
> >      - sudo apt-get install python-sphinx
> > +    - pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
>
> I don't understand. It's right here https://pypi.python.org/pypi/Cython/
> so why not just pip install the latest version?
>
> Also out of the two links, one has the Cython version 0.19.1 and the other
> link raises a 404 error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5492/files#r43788755>.
>